docs/pretrained.rst

```diff
@@ -275,11 +300,20 @@ The input output configuration is as follows:
 
     - micronet_hovernet-consep
 
+.. collapse:: Output Nuclear Classes
```

Comment:
I think micronet is just a segmentation model. No classification.

Reply:
Edited - thanks.

Reply:
Edited thanks